_rules/SC3-1-2-lang.md

```diff
@@ -1,79 +1,78 @@
 ---
-rule_id: SC3-1-2-lang
-name: Languages within the body
-test_mode: automatic
-environment: DOM Structure
+name: Validity of lang attribute value in html body elements
+
+description:
+- This rule checks that `lang` or `xml:lang` attribute on elements within the `body` of a web page conforms to [BCP 47](https://www.ietf.org/rfc/bcp/bcp47.txt).
 
 success_criterion:
-- 3.1.2 # Language of Parts (Level AA)
+- 3.1.2
+
+test_aspects:
+- DOM Tree
 
 authors:
-- Annika Nietzio
+- Bryn Anderson
+- Jey Nandakumar
 ---
 
-## Description
+## Test Procedure
 
-This test checks that `lang` attributes within the body of a web page are correct.
+### Applicability
 
-## Background
+- Any DOM element, within the `body` of a webpage with a [non-empty][] `lang` or `xml:lang` attribute.
```

Comment:
There is an inconsistent use of bullets, I am pretty sure that text blocks shouldn't be bulleted @jkodu what does the template say?